src/runtime/internal/Component.ts

```diff
@@ -22,9 +22,11 @@ interface T$$ {
 }
 
 export function bind(component, name, callback) {
-	if (component.$$.props.indexOf(name) === -1) return;
-	component.$$.bound[name] = callback;
-	callback(component.$$.ctx[name]);
+	if (name in component.$$.props) {
```

Comment:
Something I was wondering while I was writing my PR was how safe this was. It's not convenient to make this a `null`-prototype object, so maybe we ought to be using `Object.hasOwnProperty()`.

Reply:
gah. i thought this was only a thing if `Object.prototype` had been polluted with enumerable properties, but I was wrong — `'constructor' in {} === true`. fixed